src/routes/home/HomeScreen.tsx

```diff
@@ -81,9 +81,8 @@ const HomeScreen = ({ navigation }: HomeScreenProps): JSX.Element => {
 
   const selectedProfessionItems = selectedProfessions?.map(profession => (
     <DisciplineCard
-      key={profession.id}
-      discipline={profession}
-      showProgress
```

Comment:
Loading of details of a discipline is now done in the DisciplineCard, because we do no longer save the whole discipline to async storage, but only its id, so if e.g. the title or icon changes in the cms, we update it. We did not do this before, because the endpoint didn't exist.